crates/web-sys/src/features/gen_TransformStream.rs

```diff
@@ -0,0 +1,38 @@
+#![allow(unused_imports)]
+use super::*;
+use wasm_bindgen::prelude::*;
+#[wasm_bindgen]
+extern "C" {
+    # [wasm_bindgen (extends = :: js_sys :: Object , js_name = TransformStream , typescript_type = "TransformStream")]
+    #[derive(Debug, Clone, PartialEq, Eq)]
+    #[doc = "The `TransformStream` class."]
+    #[doc = ""]
+    #[doc = "[MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/API/TransformStream)"]
+    #[doc = ""]
+    #[doc = "*This API requires the following crate features to be activated: `TransformStream`*"]
+    pub type TransformStream;
+    #[wasm_bindgen(catch, constructor, js_class = "TransformStream")]
+    #[doc = "The `new TransformStream(..)` constructor, creating a new instance of `TransformStream`."]
+    #[doc = ""]
+    #[doc = "[MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/API/TransformStream)"]
+    #[doc = ""]
+    #[doc = "*This API requires the following crate features to be activated: `TransformStream`*"]
+    pub fn new() -> Result<TransformStream, JsValue>;
+    #[cfg(feature = "ReadableStream")]
+    # [wasm_bindgen (structural , method , getter , js_class = "TransformStream" , js_name = readable)]
+    #[doc = "Getter for the `readable` field of this object."]
+    #[doc = ""]
+    #[doc = "[MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/API/TransformStream)"]
+    #[doc = ""]
+    #[doc = "*This API requires the following crate features to be activated: `TransformStream`, `ReadableStream`*"]
+    pub fn readable(this: &TransformStream) -> ReadableStream;
+    #[cfg(web_sys_unstable_apis)]
```

Comment:
Omitting half the API makes the type nearly useless. Should the whole type be `unstable`?